src/ApiPort/App.config

```diff
@@ -3,12 +3,13 @@
   <startup>
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
   </startup>
-  <system.diagnostics>
-    <trace autoflush="false" indentsize="4">
+  <!-- Uncomment this to see diagnostics -->
+  <!--<system.diagnostics>
+    <trace autoflush="true" indentsize="4">
       <listeners>
-        <add name="myListener" type="System.Diagnostics.ConsoleTraceListener" />
+        <add name="logListener" type="System.Diagnostics.TextWriterTraceListener" initializeData="ApiPort.log" />
         <remove name="Default" />
       </listeners>
     </trace>
-  </system.diagnostics>
+  </system.diagnostics>-->
 </configuration>
```

Comment:
Currently, it spews it out for everyone, when the messaging should let you know what the issue is. This change makes it so it logs it to a file and only when you explicitly enable it. I'm fine with enabling it by default, I just don't want it output to the console all the time.